src/CashFlows.py

```diff
@@ -109,6 +113,7 @@ def __init__(self, verbosity=100, **kwargs):
     self._activeComponents = None
     self._metricTarget = None
     self._components = []
+    self._outputtype = None
```

Comment:
this should probably be camel back, as `self._outputType`. Appears in several places.